### PR TITLE
SC-3135: Features and improvements

### PR DESCRIPTION
**Pull request by sprymiker**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/SC-3135

#### Optional

- Suite PR: https://github.com/spryker/suite-nonsplit/pull/4210

### CODE REVIEW

File: bin/registry/help.sh [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        local -i padLength=$((padding - len))
+        local pad=$(printf "%-${padLength}.${padLength}s" ' ')
+
+        echo -e "$col0 $col1${pad}  $col2"
+    done
+
+    return "${TRUE}"
+}
+
+#"
```
Comment (zyuzka) [inline bin/registry/help.sh:92]: should be removed?

Comment (sprymiker) [reply bin/registry/help.sh]: Done.

File: bin/sdk/images/baked.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import sdk/images/common.sh
+
+function Images::build() {
+    for arg in "${@}"; do
+        case "${arg}" in
+            '--force')
+                # it is always it.
+                ;;
+            '--no-cache')
```
Comment (zyuzka) [inline bin/sdk/images/baked.sh:11]: tech dept?

File: bin/sdk/images/mount.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import sdk/images/common.sh
+
+function Images::build() {
+    for arg in "${@}"; do
+        case "${arg}" in
+            '--force')
+                # it is always it.
+                ;;
+            '--no-cache')
```
Comment (zyuzka) [inline bin/sdk/images/mount.sh:11]: same

File: bin/command/build/baked/export.sh [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    subCommand=${1}
+    shift || true
+
+    while getopts "t:p:" opt; do
+        case "${opt}" in
+            t)
+                tag=${OPTARG}
+                ;;
+            p)
+                # Deprecated
```
[the remark this answers is not in the document]
Comment (sprymiker) [reply bin/command/build/baked/export.sh:29]: This option is used for `docker/sdk export assets` that is deprecated.

File: bin/sdk/images/baked.sh (hunk 2 above)
Comment (sprymiker) [reply bin/sdk/images/baked.sh:11]: Kind off. Dummies for future improvements.

File: bin/sdk/images/mount.sh (hunk 3 above)
Comment (sprymiker) [reply bin/sdk/images/mount.sh:11]: Dummies for future improvements.

**Pull request merged**